docs/guide/README.md

```diff
@@ -1,94 +1,94 @@
+## How it connects with backend platforms?
 
+Vue Storefront manages to be platform agnostic thanks to the [vue-storefront-api](https://github.com/DivanteLtd/vue-storefront-api) and dedicated API connectors for eCommerce backend platforms. The data format in vue-storefront-api is always the same for any platform which means no matter which eCommerce backend you'll use your frontend remains without changes.
+
+It's a great strategy for migrations since you can easily migrate from one platform to another (or another version like Magento 1 -> 2) without touching your frontend.
 
 ![Architecture diagram](https://raw.githubusercontent.com/DivanteLtd/vue-storefront/master/docs/.vuepress/public/GitHub-Architecture-VS.png)
 
 The API connector works in two phases:
-- **data pump** (on the image mage2nosql)  is pulling static data (catalog, orders, etc.) from your eCommerce platform to Vue Storefront Elasticsearch and changes its format to the one consumed by vue-storefront-api. After pulling the data, you can display the product catalog in Vue Storefront. After pumping the data into Elasticsearch, it will stay in sync with changes on the backend platform side and update its content.
-
-- **worker pool** is a synchronization of so-called dynamic calls (user sessions, cart rules, etc.) that can't be stored in the database and need to be called by vue-storefront-api directly from the backend platform.
 
-While managing these two phases of integration, Vue Storefront can work with your backend platform.
+- **data pump** (on the image mage2nosql) is pulling static data (catalog, orders etc) from your eCommerce platform to Vue Storefront ElasticSearch and changes it's format to the one consumed by vue-storefront-api. After pulling the data you can display product catalog in Vue Storefront. After pumping the data into ElasticSearch it will stay in sync with changes on the backend platform side and update its content.
+- **worker pool** is a synchronization of so-called dynamic calls (user sessions, cart rules etc) that can't be stored in the database and needs to be called by vue-storefront-api directly from the backend platform.
 
+While managing these two phases of integration Vue Storefront can work with your backend platform.
 
-Some of the backend platforms already have their integrations (Magento 2, Magento 1, CoreShop, BigCommerce, WooCommerce), but you can easily make your own with the [integration boilerplate](https://github.com/DivanteLtd/bigcommerce2vuestorefront
-).
+Some of the backend platforms already have their integrations (Magento 2, Magento 1, CoreShop, BigCommerce, WooCommerce) but you can easily make your own with [integration boilerplate](https://github.com/DivanteLtd/bigcommerce2vuestorefront).
 
 The blue parts on the diagram are responsible for offline cache and will be explained later in the article.
 
-## How does it work?
-
-There are three concepts you need to be familiar with while working with Vue Storefront.
+## How it works?
```

Comment:
"How does it work?" was correct english, "How it works?" is not. Please change back.